providers/flagd/pkg/provider.go

```diff
@@ -120,22 +131,57 @@ func FromEnv() ProviderOption {
 			p.providerConfiguration.SocketPath = socketPath
 		}
 
-		cachingDisabled := os.Getenv(flagdCachingDisabledEnvironmentVariableName)
-		if cachingDisabled == "true" {
-			WithoutCache()(p)
-		}
-
 		certificatePath := os.Getenv(flagdServerCertPathEnvironmentVariableName)
 		if certificatePath != "" || os.Getenv(flagdTLSEnvironmentVariableName) == "true" {
 			WithTLS(certificatePath)(p)
 		}
+
+		maxCacheSizeS := os.Getenv(flagdMaxCacheSizeEnvironmentVariableName)
+		if maxCacheSizeS != "" {
+			maxCacheSizeFromEnv, err := strconv.Atoi(maxCacheSizeS)
+			if err != nil {
+				p.logger.Error(err, fmt.Sprintf("invalid env config for %s provided, using default value", flagdMaxCacheSizeEnvironmentVariableName))
```

Comment:
Minor - should we state the default value in the log 🤔  ?

Reply:
Good idea, done